src/understanding-deis/concepts.md

```diff
@@ -11,27 +11,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
```

Comment:
Yeah. I'm building thoughts on the overall structure of the docs site, but since we have so many folks currently working on docs I don't want to create an unmergeable mess!

Once things settle down, we'll wiggle a bunch of stuff into place right before we finalize the docs for beta.